jvb/src/main/java/org/jitsi/videobridge/stats/VideobridgeStatistics.java

```diff
@@ -197,15 +191,20 @@ private void generate0()
         int octoConferences = 0;
         int endpoints = 0;
         int octoEndpoints = 0;
-        double fractionLostSum = 0d; // TODO verify
-        int fractionLostCount = 0;
-        long packetsReceived = 0;
-        long packetsReceivedLost = 0; // TODO verify
-        long bitrateDownloadBps = 0;
-        long bitrateUploadBps = 0;
+        double bitrateDownloadBps = 0;
+        double bitrateUploadBps = 0;
         long packetRateUpload = 0;
         long packetRateDownload = 0;
 
+        // Packets we received
+        long packetsReceivedReceived = 0;
```

Comment:
How about `incomingPacketsReceived`, `incomingPacketsLost`, `outgoingPacketsReceived`, and `outgoingPacketsLost`?